jarviscli/packages/timeIn.py

```diff
@@ -27,11 +39,22 @@ def main(self, s):
 
 
 def getLocation(s):
-    send_url = ("http://maps.googleapis.com/maps/api/geocode/json?address={0}&sensor=false".format(s)
-                )
+    file_path = module_path + '/../data/keys.json'
+    with open(file_path) as json_file:
+        data = json.load(json_file)
+    key = data['timein']
+    send_url = ("https://maps.googleapis.com/maps/api/geocode/json?address={0}&key={1}".format(s, key))
+    # https://developers.google.com/maps/documentation/geocoding/start?hl=en_US
+    # https://maps.googleapis.com/maps/api/geocode/json?address=1600+Amphitheatre+Parkway,+Mountain+View,+CA&key=YOUR_API_KEY
     r = requests.get(send_url)
     j = json.loads(r.text)
-    lat = j['results'][0]['geometry']['location']['lat']    # Latitude
-    lng = j['results'][0]['geometry']['location']['lng']    # Longitude
-    # Returns both latitude and longitude as a tupple
-    return lat, lng
+    try:
+        lat = j['results'][0]['geometry']['location']['lat']    # Latitude
+        lng = j['results'][0]['geometry']['location']['lng']    # Longitude
+        # Returns both latitude and longitude as a tupple
+        return lat, lng
+    except IndexError:
```

Comment:
```suggestion
    except IndexError r:
```
Actually very reasonable asking to open an github issue instead hard of crashing ;). But suppose it would be better to to try-except every Error in CmdInterpreter before calling any command.

Reply:
Forget this one ;). I'll create a pull request some day, because it's a different topic.